.circleci/config.yml

```diff
@@ -396,6 +396,33 @@ jobs:
             - docs/docs
             - docs/index.html
 
+  Publish Rust crates:
+    docker:
+      - image: circleci/rust:latest
+    steps:
+      - checkout
+      - run:
+          name: Publish Cargo Package
+          command: |
+            # Login to crates.io so the following commands work
+            cargo login -- "$CRATES_IO_TOKEN"
+
+            # Publish all crates from CI.
+            # The token is set in CircleCI settings.
+            # Need some sleep inbetween to ensure crates.io is updated.
+
+            pushd glean-core
+            cargo publish --verbose
+
+            sleep 30
```

Comment:
This is ... interesting.  I think I understand why having done this manually.  But maybe something that repeatedly tried to `curl` the new package would be more robust?

Reply:
yeah, it's ... a hack.

`cargo-release` for example [waits for the index to update](https://github.com/sunng87/cargo-release/blob/f4400027579a795e790b3885e33228638a6b111a/src/cargo.rs#L59), but of course that could happen due to other crates as well, [so they still sleep](https://github.com/sunng87/cargo-release/commit/58f7c28f43a20e3df128e165fbbfa9406d519d4b).

IMO we can go with this solution for now and, if it fails to often, fix it better.